ssf/samples.go

```diff
@@ -22,6 +33,19 @@ func Timestamp(ts time.Time) SampleOption {
 	}
 }
 
+// SampleRate sets the rate at which a measurement is sampled. The
+// rate is a number on the interval (0..1] (1 means that the value is
+// not sampled). Any numbers outside this interval result in no change
+// to the sample rate (by default, all SSFSamples created with the
+// helpers in this package have a SampleRate=1).
+func SampleRate(rate float32) SampleOption {
```

Comment:
At this point, if we're only introducing a single `SampleOption` function, can we just replace it with a raw value?

Reply:
There's multiple others in that file that are used (:

Reply:
Ah yes, not in the diff. Carry on!